plutus-core-spec/figures/PlutusCoreGrammar-Algorithmic-Restricted.tex

```diff
@@ -0,0 +1,51 @@
+\documentclass[../main.tex]{subfiles}
```

Comment:
I'm not wild about just duplicating this whole thing. Can't we describe this as just a diff to the alternative version or something?

Reply:
Similarly for the others. For a reader as well it is very unclear where the differences are.

Reply:
What sort of annotation would be useful? There are a number of differences in various places at different levels. Like, I feel it's useful to know which rules change, but also to know which parts? But the parts have different scales (sometimes whole premises are added or removed, sometimes a metavar name is changed). I don't want to add too much visual clutter. Do you have any suggestions?

Reply:
I suggested in another place, but maybe a background tint like Pierce does for the successive systems in TAPL?

Reply:
I'll take a look. I feel like it'd be important to say both "this rule was changed" and also "this is the part that we changed" but that might be very cluttered??